t/10-jobs.t

```diff
@@ -182,6 +181,22 @@ subtest 'job with at least one softfailed and rest passed => overall is softfail
     is($job->skipped_module_count,    0,                                   'number of skipped modules not incremented');
 };
 
+subtest 'inserting the same module twice is not messing the job module statistics' => sub {
```

Comment:
can we find a positive wording for "not messing"? Like "keeps the job module statistics intact"?